Scripts/PowerShell/Get-DeviceList.ps1

```diff
@@ -41,38 +44,125 @@ limitations under the License.
    .\Get-DeviceList.ps1 -IpAddress "10.xx.xx.xx"
    In this instance you will be prompted for credentials to use
 #>
+
 [CmdletBinding()]
 param(
     [Parameter(Mandatory)]
     [System.Net.IPAddress] $IpAddress,
 
+    [Parameter(Mandatory)]
+    [String] $Fileextension,
```

Comment:
Can we specify an output format and limit choices to csv / json - also these two parameters (output format and filepath) should not be Mandatory - if nothing is specified for output format then it defaults to current behavior which is printing out JSON to stdout